empty/.post-create

```diff
@@ -0,0 +1,25 @@
+To begin development, install the Temporal CLI:
+
+  Mac: {cyan brew install temporal}
+  Other: Download and extract the latest release from https://github.com/temporalio/cli/releases/latest
+
+Start Temporal Server:
+
+  {cyan temporal server start-dev}
```

Comment:
I would recommend adding `--db-filename clusterdata.db` to this command or at least mentioning that this is how we would get Workflows to persist so in the case if the cluster crashed, or if the user shut it down before a break and started it up later, all of their Workflow Executions (and any other data) would be maintained